src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -3572,10 +3465,13 @@ public enum Platinum implements LogicCardInfo {
             }
```

Comment:
Not directly related to the fix, but the extraPoison is being considered attack-sourced. So cards like Bibarel (Stormfront), who should only prevent effects from attacks block the additional poison counters.